### ISSUE TITLE
While resizing, a white window background is shown

### ISSUE THREAD

**Issue by Julesssss** [Expensify/App#13084]
### Problem 

When resizing the web window in dark mode there's a very obvious white color as the window background. 

https://user-images.githubusercontent.com/10736861/203777318-e21cb0bf-acef-4c3b-bc91-b6346e8fd7fd.mov

_This was pointed out [here on the PR](https://github.com/Expensify/App/pull/12944#pullrequestreview-1193032363), but made a separate issue to help unblock the merge._ 

### Solution

We should try to use our bg colour instead.

**Comment by Julesssss** [comment]
Replacing[ this issue](https://github.com/Expensify/App/issues/13085) with this one to enable the full checklist and bugzero process
